nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -91,14 +91,38 @@ The following changes have been made to the `Prefix` model.
 |------------------------|-----------------|
 | `get_child_prefixes()` | `descendants()` |
 
-#### IPAddressd Parenting Concrete Relationship
+### IPAddress Parenting Concrete Relationship
 
 The `ipam.IPAddress` model has been modified to have a foreign key to `ipam.Prefix` as the `parent` field. Parenting of IP addresses is now automatically managed at the database level to greatly improve performance especially when calculating tree hierarchy and utilization.
 
 | Removed                | Replaced With   |
 |------------------------|-----------------|
 | `get_child_prefixes()` | `descendants()` |
 
+### Prefix get_utilization Method
```

Comment:
Would it make sense to duplicate some/all of this section into `nautobot/docs/models/ipam/prefix.md`?